### PR TITLE
feat: add "GET /services"

### LINKED ISSUE #1: Implement "GET /services"

### PR DESCRIPTION
**Pull request by ninsch3000**
* add `getServices` controller for endpoint `GET /services`

Resolves #1

### CODE REVIEW

File: tests/ga4gh/registry/test_server.py [hunk 1]
```diff
@@ -29,9 +30,18 @@ def test_getServices():
...
     )
+    mock_resp = deepcopy(MOCK_SERVICE)
+    mock_resp['id'] = MOCK_ID
+    app.config['FOCA'].db.dbs['serviceStore'].collections['services'] \
+        .client = mongomock.MongoClient().db.collection
+    app.config['FOCA'].db.dbs['serviceStore'].collections['services'] \
+        .client.insert_one(mock_resp)
+
+    data = deepcopy(MOCK_SERVICE)
+    data['id'] = MOCK_ID
```
Comment (ninsch3000) [inline tests/ga4gh/registry/test_server.py]: Might make sense to add another assert, where service list retrieval independent of ID is ensured?

File: tests/mock_data.py [hunk 2]
```diff
@@ -72,3 +72,9 @@
         "service-info"
     )
 }
+MOCK_SERVICE = {
```
Comment (ninsch3000) [inline tests/mock_data.py:75]: Included required properties from [ga4gh schema](https://raw.githubusercontent.com/ga4gh-discovery/ga4gh-service-info/v1.0.0/service-info.yaml#/components/schemas/Service), analogous to trs_filer I left out `id` and am setting it during the test.

File: cloud_registry/ga4gh/registry/server.py [hunk 3]
```diff
@@ -3,7 +3,7 @@
 import logging
 from typing import (Dict, List, Tuple)
 
-from flask import request
+from flask import (request, current_app)
```
Comment (uniqueg) [inline cloud_registry/ga4gh/registry/server.py]: Better in alphabetical order, so `(current_app, request)`

File: cloud_registry/ga4gh/registry/server.py [hunk 4]
```diff
@@ -15,8 +15,20 @@
 
 @log_traffic
 def getServices() -> List:
-    """"""
-    return []
+    """List all services
```
Comment (uniqueg) [inline cloud_registry/ga4gh/registry/server.py]: Please add a full stop after synopsis.

File: tests/mock_data.py (hunk 2 above)
Comment (uniqueg) [reply tests/mock_data.py:75]: Or actually you _could_ add `id`, since you're not using it in the other test anyway

File: tests/ga4gh/registry/test_server.py (hunk 1 above)
Comment (uniqueg) [reply tests/ga4gh/registry/test_server.py]: You could assert for list length of `1` in your case. You could also add a test where you _don't_ have any services added to check against the empty list. Or possibly you can add this assert to _this_ test by adding a `with app.app_context()` section before you call `.insert_one(mock_resp)`. But might still require to set a mock response (to `[]`), not sure.

**Review by uniqueg — CHANGES_REQUESTED** [review]

File: cloud_registry/exceptions.py [hunk 5]
```diff
@@ -50,7 +50,7 @@
         "status": 404,
     },
     InternalServerError: {
-        "title": "Internal server errro",
```
Comment (uniqueg) [inline cloud_registry/exceptions.py:53]: Thanks!

File: cloud_registry/ga4gh/registry/server.py [hunk 6]
```diff
@@ -1,9 +1,9 @@
-"""Controllers for DRS endpoints."""
+"""Controllers for service endpoints."""
```
Comment (uniqueg) [inline cloud_registry/ga4gh/registry/server.py:1]: Thanks! :)

**Review by uniqueg — APPROVED** [review]
Thanks 🙏🏻
